CHANGELOG_CLOUD.md

```diff
@@ -0,0 +1,688 @@
+# Changelog
+All notable changes to this project for *Dgraph's cloud releases* will be documented in this file.
+The v23.0.0 release combines the open source and cloud releases and we won't have any
+separate cloud releases going forward.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+
+## [v23.0.0-beta1] - 2021-10-30
+[v23.0.0-beta1]: https://github.com/dgraph-io/dgraph/compare/v22.0.2...v23.0.0-beta1
```

Comment:
this diff should be between `21.03-92` vs `v23.0.0-beta1`

Reply:
https://github.com/dgraph-io/dgraph/compare/v21.03.0-92-g0c9f60156...v23.0.0-beta1 <- url

Reply:
```
fix(parser): Check order of inputs before evaluating nested eq func (#8679)
    fix(query): handle bad timezone correctly (#8657)
    fix(zero): fix waiting for random time while rate limiting (#8656)
    fix(proposals): Incremental proposal key for zero proposals (#8005) (#8567)
    fix(probe): do not contend for lock in lazy load (#8037) (#8041) (#8566)
    fix(restore): Set kv version to restoreTs for all keys (#7930) (#8563)
    fix(groot): do not upsert groot for all namespaces on restart (#8561)
    fix(mutation): validate mutation before applying it (#8623)
    fix(backup): create directory before writing backup (#8638)
    fix(restore): consider the banned namespaces while bumping (#7839) (#8559)
    fix(backup): use StreamWriter instead of KVLoader during backup restore (#8510)
    fix(GraphQL): Nested Auth Rules not working properly. (#7915) (#8084) (#8571)
    fix(zero): fix update membership to make bulk tablet proposal instead of multiple small (#8090) (#8573)
    fix(debug): Check length of wal entry before parsing (#8560)
    fix(GraphQL): Pass on HTTP request headers for subscriptions (#7806) (#7807) (#8574)
     - `fix(Query)`: Skipping floats that cannot be Marshalled in JSON
    fix(pagination): Fix after for regexp, match functions (#7700) (#8471)
    fix(DQL): optimize query for has function with offset. (#7727) (#8431)
    fix(sort): Only filter out nodes with positive offsets. (#8077) (#8441)
    fix(query): Do not execute filters if there are no source uids (#7962… (#8452)
    fix(fragment): merge the nested fragments fields (#8075) (#8435)
     - `fix(Query)`: Skipping floats that cannot be Marshalled in JSON
    fix(admin): make config changes to pass through gog middlewares (#8014) (#8442)
    fix(ACL): Prevents permissions overrride and merges acl cache to persist permissions across different namespaces (#8418)
     - `fix(Query)`: Skipping floats that cannot be Marshalled in JSON
    fix(deps): update dependencies on badger + ristretto (#8365)
    fix(GraphQL): optimize eq filter queries (#7895) (#8334)
     - `fix(Query)`: Skipping floats that cannot be Marshalled in JSON
    fix(GraphQL): add validation of null values with correct order of graphql rule validation (#8333)
     - `fix(Query)`: Skipping floats that cannot be Marshalled in JSON
    fix(query): Prevent multiple entries for same predicate in mutations.2022 (#8332)
     - `fix(Query)`: Skipping floats that cannot be Marshalled in JSON
    fix(GraphQL) fix auth query rewriting with ID filter (cherry-pick-7740) (#8157)
    fix(rollups): cherry-pick-7609 (#8297)
    fix(bug) : use copy of loop variable in Go routines (#8163)
    fix(docs): fix badges  (#8291)
    fix(live): quote the xid when doing upsert (#7985) (#7999)
    * fix(acl): subscribe for the correct predicates (#7992)
    fix(acl): filter out the results based on type (#7978) (#7981)
    fix(pool): use write lock when getting health info (#7963) (#7967)
    fix(conn): JoinCluster loop should use latest conn (#7950) (#7952)
    fix(backup): Fix full backup request (#7932) (#7934)
    fix(Raft): Detect network partition when streaming (#7908)
    fix(metrics): Expose dgraph_num_backups_failed_total metric view. (#7900) (#7904)
    fix(DQL): revert changes related to cascade pagination with sort (#7885) (#7888)
    fix(restore): append galaxy namespace to type name (#7881)
    fix(admin): remove exportedFiles field (#7835) (#7836)
    fix(ee): GetKeys should return an error (#7713) (#7797)
    cloud: fix(auth): preserve the status code while returning error (#7832) (#7834)
    * fix(auth): preserve the status code while returning error (#7832)
    cloud: fix(lease): prevent ID lease overflow (#7802)
    * fix(lease): prevent ID lease overflow (#7724)
    * fix(live): make live loader progress on a cluster with very high maxUid  (#7743)
    fix(lease): don't do rate limiting when not limit is not specified (#7787)
    fix(export): Fix facet export of reference type postings to JSON format (#7744) (#7756)
    fix(schema-update): Start opIndexing only when index creation is required. (#7845) (#7847)
    fix(drop): attach galaxy namespace to drop attr done on 20.11 backup (#7827)
    fix(txn): ensure that txn hash is set (#7782) (#7784)
    fix(bulk): upsert guardian/groot for all existing namespaces (#7759) (#7769)
    fix(raftwal): take snapshot after restore (#7719) (#7750)
    fix(bulk): throw the error instead of crashing (#7722) (#7749)
    fix(Chunker): don't delete node with empty facet in mutation (#7737) (#7745)
```

Reply:
Done.